packages/components/src/__future__/Tag/Tag.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useMemo, FC, ReactNode, Children, isValidElement } from "react"
+import classNames from "classnames"
+import { v4 } from "uuid"
+
+const isElementText = (child: ReactNode): boolean =>
+  typeof child === "string" && child.replace(/\s/g, "").length > 0
+
+const isElementNumber = (child: ReactNode): boolean => typeof child === "number"
+
+const isElementIcon = (child: ReactNode): boolean =>
+  isValidElement(child) &&
+  typeof child.type !== "undefined" &&
+  typeof child.type !== "string" &&
+  typeof child.type !== "number" &&
+  child.type.name === "Icon"
+
+type TagColors =
+  | "gray"
+  | "blue"
+  | "green"
+  | "yellow"
+  | "orange"
+  | "red"
+  | "purple"
+
+export interface TagProps {
+  children: ReactNode
+  color?: TagColors
+}
+
+export const Tag: FC<TagProps> = ({ children, color = "gray" }) => {
+  const uuid = useMemo(() => v4(), [])
+
+  return (
+    <span
+      className={classNames(
+        "inline-flex items-center px-8 py-4 gap-4 rounded-[18px] font-family-paragraph text-paragraph-sm font-weight-paragraph leading-paragraph-sm whitespace-nowrap",
+        {
+          [`bg-${color}-100`]: color !== "gray",
+          [`text-${color}-700`]: color !== "gray",
+          ["bg-gray-200"]: color === "gray",
+          ["text-purple-800"]: color === "gray",
+        }
```

Comment:
String interpolation doesn't work in tailwind, it only works here because of our storybook setup. We will take a look at making this more closer to production so we don't have this magically working.